WordPress/Classes/ViewRelated/NUX/NUXAbstractViewController.swift

```diff
@@ -17,6 +17,8 @@ class NUXAbstractViewController : UIViewController
     let helpBadgeSize = CGSize(width: 12, height: 10)
     let helpButtonContainerFrame = CGRect(x: 0, y: 0, width: 44, height: 44)
 
+    var dismissBlock: ((cancelled: Bool) -> Void)?
```

Comment:
I think I'd want to reverse this and have it be something like `completionBlock: ((success: Bool) -> Void)?`